user/enterprise/build-images.md

```diff
@@ -93,7 +95,34 @@ After that, both Docker and travis-worker have to be restarted. For Docker, plea
 $ sudo restart docker
 ```
 
-For travis-worker, you can find the instructions [here](https://docs.travis-ci.com/user/enterprise/worker-cli-commands/#Stopping-and-Starting-the-Worker).
+__Ubuntu 16.04 as Host operating system__
+
+To have the Docker daemon listening to the private IP address, please open `/etc/docker/daemon.json`. In there, please configure `hosts` like this:
+
+```json
+{
+    //...
+    "hosts": [
+        "fd://",
+        "tcp://172.31.7.199:4243"
```

Comment:
is this a specific address? Or is there a way to tell the user what this might look like on their machine? I'm guessing the thing we're adding here is the `fd://`? Or is it this address?

Reply:
@bnferguson @plaindocs Ah good catch with that `fd://`. This is the new way how Docker is running on the machine where before it was a domain socket but now it's socket activation. I'm wondering if we should explain that since it's fairly new and different from the current way of installing and configuring Enterprise or if we can expect the reader to know what `fd://` stands for.